cfn-resources/serverless-instance/docs/README.md

```diff
@@ -151,11 +143,17 @@ _Type_: Boolean
 
 _Update requires_: [No interruption](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-updating-stacks-update-behaviors.html#update-no-interrupt)
 
-## Return Values
+#### Profile
+
+Profile used to provide credentials information, (a secret with the cfn/profile/{Profile_name}, is required), if not provided Default is used
```

Comment:
```suggestion
Profile used to provide credentials information, (a secret with the cfn/profile/{Profile_name}, is required), if not provided 'cfn/atlas/profile/default' is used
```

Reply:
[q] Can we add documentation about how to create this profile with AWS secret manager?